### PR TITLE
Update S & T checkboxes.html

### PR DESCRIPTION
**Pull request by Shravanikale**
added the missing programming languages starting from alphabets S and T with the help of given resources provided. 
i.e., `https://en.wikipedia.org/wiki/List_of_programming_languages`

### CODE REVIEW

File: A-Z-checkboxes.html [hunk 1]
```diff
@@ -1554,9 +1725,59 @@ <h1>S</h1>
...
+<input type="checkbox" id="ttcn" name="pchbx" value="ttcn" />
+<label for="trac"> TTCN </label><br />
+
+<input type="checkbox" id="tutor" name="pchbx" value="tutor" />
+<label for="tutor"> TUTOR </label><br />
+
+<input type="checkbox" id="txl" name="pchbx" value="txl" />
+<label for="txl"> TXL </label><br />
 
-<input type="checkbox" id="typescript" name="pchbx" value="typescript" />
```
Comment (chilarai) [inline A-Z-checkboxes.html:1558]: Why did you remove Typescript?

**Review by chilarai — CHANGES_REQUESTED** [review]
Thanks for the PR. But please resolve one issue

**Comment by Shravanikale** [comment]
done with the changes !!
